### PR TITLE
Small changes/additions for Dalitz, from Edgar

### PR DESCRIPTION
**Pull request by amaringarcia**

### CODE REVIEW

File: PWGGA/GammaConv/macros/AddTask_GammaConvDalitzV1_pp.C [hunk 1]
```diff
@@ -1,6 +1,6 @@
 
 void AddTask_GammaConvDalitzV1_pp(  Int_t trainConfig = 1,  //change different set of cuts
-                                    Bool_t isMC   = kFALSE, //run MC
+                                    Bool_t isMC   = kFALSE, //0 Exp data, 1 MC data, 2 JJ MC for improves.
```
Comment (nschmidtALICE) [inline PWGGA/GammaConv/macros/AddTask_GammaConvDalitzV1_pp.C]: The comment makes no sense if the variable is a boolean. Do you want to change this to an integer?

**Comment by amaringarcia** [comment]
For the first commit I will need a change that Edgar is not ready with. So the second commit changes only the numbers but let the definition untouched. On 2 Dec 2019 4:00 pm, nschmidtALICE <notifications@github.com> wrote:@nschmidtALICE commented on this pull request.



In PWGGA/GammaConv/macros/AddTask_GammaConvDalitzV1_pp.C:
> @@ -1,6 +1,6 @@
 
 void AddTask_GammaConvDalitzV1_pp(  Int_t trainConfig = 1,  //change different set of cuts
-                                    Bool_t isMC   = kFALSE, //run MC
+                                    Bool_t isMC   = kFALSE, //0 Exp data, 1 MC data, 2 JJ MC for improves.

The comment makes no sense if the variable is a boolean. Do you want to change this to an integer?

**Comment by nschmidtALICE** [comment]
+1

**Comment by nschmidtALICE** [comment]
Then i guess the features requiring the integer will be implemented soon.

**Pull request merged**